scripts/zones/Empyreal_Paradox/mobs/Selhteus.lua

```diff
@@ -11,6 +11,7 @@ local entity = {}
 
 entity.onMobInitialize = function(mob)
     mob:addMod(xi.mod.REGAIN, 50)
+    mob:addMod(xi.mod.CURE_POTENCY_RCVD, -100)
```

Comment:
is he targettable?

Reply:
my swiss cheese memory has to legit ask ._.

Reply:
Resolved via DMs

Reply:
> [5:33 PM] TeoTwawki:
>>    mob:addMod(xi.mod.CURE_POTENCY_RCVD, -100)
>
>Sehlteus - iirc we can't even target him on retail tho.
>
> [5:34 PM] Aerec: Uh I just watched videos and noone tried to target him.
[5:34 PM] Aerec: I have someone going through that fight on retail soon
[5:34 PM] Aerec: so well have a definite answer on that in like a week or two
[5:34 PM] Aerec: its a one line change tho
[5:35 PM] TeoTwawki: kk
[5:35 PM] Aerec: That was a 'best' guess.
[5:35 PM] Aerec: I originally had, setUntargetable.
[5:35 PM] Aerec: But I couldn't really find anything that said
[5:35 PM] Aerec: You can't target him
[5:35 PM] Aerec: So I just set him to have cures not do anything.
[5:35 PM] Aerec: since everything says cures don't do anything to him
[5:36 PM] Aerec: ¯\_(ツ)_/¯
[5:37 PM] TeoTwawki: -nod-
[5:39 PM] Aerec: So that was one of the: As soon as I have the exact answer its a one line fix later.
[5:39 PM] Aerec: But for now, it matches all acceptable criteria imo

ima hold you to that now that its merged